bedrock/contentful/api.py

```diff
@@ -748,7 +751,7 @@ def get_media_class():
             return " ".join(media_classes)
 
         def get_mobile_class():
-            mobile_display = fields.get("mobile_display")
+            mobile_display = fields.get("mobileDisplay")
```

Comment:
Let's revert this change for now. I'd consider this fix out of scope for the current task and have added a separate issue to [address testing the Split Contentful component and fixing bugs where necessary](https://github.com/mozilla/bedrock/issues/11766).

Running
`print(fields.get("mobile_display"))`
`print("Center content" in mobile_display)`
Outputs
```
[0] ['Center conent']
[0] False
```

The typo is in the option value: "Center conent" should be "Center content"

If I fix option value in Contentful model, the mobile centering class gets doubled-up (given the hardcoded `mzp-l-split-center-on-sm-md` default above)
<img width="677" alt="redundant-class" src="https://user-images.githubusercontent.com/19650432/173350618-97f3a4af-e50a-4ee7-b7f2-733150b2e732.png">

Since the bug wasn't introduced here and is a bit more complex than originally thought, let's leave in original state and fix in separate issue with tests.

Reply:
Sounds good to me, should be an easier fix since it originated in Contentful.